examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -90,6 +92,13 @@ int AppTask::Init()
         return ret;
     }
 
+    ret = sNFC.Init(ConnectivityMgr());
```

Comment:
Can we make some "ifdef" e.g. sth like `#ifdef CHIP_NFC_COMMISSIONING`.
The reason behind is that this won't be supported on nRF52840 Dongle, which we plan to use as well soon.

We can also do that in a subsequent PR.